components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -4174,8 +4174,10 @@ private static String getSPJwksUrl(String clientId, String spTenantDomain) throw
             }
         }
         if (log.isDebugEnabled()) {
-            log.debug(String.format("Retrieved jwks uri: %s for the service provider associated with client_id: %s",
-                    jwksUri, clientId));
+            if (StringUtils.isNotBlank(jwksUri)) {
```

Comment:
If the jwksUri is empty then we can print it as empty right? Any reason to neglect debug message if jwksUri is empty?

Reply:
If jwksUri is empty, then it will give a debug message after the getSPJwksUrl method calls (Example - [OAuth2Util.java#L2482](https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java#L2482)). 
So checking jwksUri is not empty inside the getSPJwksUrl method to overcome the redundancy.